docs/source/workflows/descriptors.rst

```diff
@@ -63,13 +63,11 @@ A set of descriptors defines a controlled vocabulary with which to describe AI t
 The :class:`~citrine.builders.descriptors.PlatformVocabulary` class is provided to collect a set of descriptors,
 associate them with short convenient names, and provide them via a familiar dictionary interface.
 
-While descriptors cannot be independently saved on the platform for reuse, AttributeTemplates can be.
-Therefore, common descriptors can be saved as AttributeTemplates to the data platform, effectively sharing them with
-other users.
-The `PlatformVocabulary.from_templates` method facilitates this pattern by automatically downloading AttributeTemplates
-and converting them into descriptors.
-AttributeTemplates must be associated with a namespace via Alternative Identifiers (the `uids` field).
-When calling `from_templates`, a scope is provided to select one of those namespaces.
+While descriptors cannot be independently saved on the platform for reuse, :class:`~citrine.resources.attribute_templates.AttributeTemplate`s can be.
+Therefore, common descriptors can be saved as attribute templates to the data platform, effectively sharing them with other users.
+:meth:`~citrine.builders.descriptors.PlatformVocabulary.from_templates` facilitates this pattern by automatically downloading attribute templates and converting them into descriptors.
```

Comment:
This link to the `from_templates` method doesn't resolve for me when I build the docs locally. Could someone else try it and see if it's just by computer (and if not, figure out why it doesn't resolve)?

Reply:
The backtick-s combo seems to be leaking some state, interfering with the next render.  The suggestion above repairs this behavior (though I don't really know why).